src/webview/js/js.js

```diff
@@ -679,9 +683,10 @@ export const handleInitialLoad = (
   // The `realm` part of an `Auth` object is a URL object. It's passed
   // in its stringified form.
   rawAuth: {| ...$Diff<Auth, {| realm: mixed |}>, realm: string |},
+  doNotMarkMessagesAsRead: boolean,
 ) => {
   const auth: Auth = { ...rawAuth, realm: new URL(rawAuth.realm) };
-
+  doNotMarkMessagesAsReadFlag = doNotMarkMessagesAsRead;
   // Since its version 5.x, the `react-native-webview` library dispatches our
```

Comment:
nit: keep the blank line -- it's serving a purpose by separating this comment from what's above, so that it's visually attached to the whole stanza below